src/mudlet.h

```diff
@@ -432,6 +430,9 @@ class mudlet : public QMainWindow, public Ui::main_window
     // Options dialog when there's no active host
     QPointer<dlgProfilePreferences> mpDlgProfilePreferences;
 
+    static void setMirrorState(const bool state) { mMirrorToStdOut = state; }
```

Comment:
:facepalm: Oh, I've just spotted that since the variable itself is now public and `static` (and setting its value does not require any "processing") we might just as well use it directly as the "global" flag that it is... i.e. just refer to it directly in other classes as `mudlet::mMirrorToOut` - the only risk is if it accidentally gets modified when it is meant to be just tested against (i.e. with something like `if (mudlet::mMirrorToOut = true)`instead of `if (mudlet::mMirrorToOut == true)` - OTOH as a `bool` value one is not likely to be testing the value with a `==` anyhow!)

Reply:
Right, and we have clang-tidy to warn us here of that

Reply:
So is that worth doing - i.e. dispensing with `(void) setMirrorState(const bool)` and `(bool) getMirrorState()` and using the raw `mudlet::mMirrorToStdOut` directly?

Reply:
Yep, I'll do it